app/views/wiki/revisions.html.erb

```diff
@@ -1,79 +1,81 @@
+          | <%= time_ago_in_words(revision.created_at) %> ago
+          <div style="display:none;" id="body_<%= @node.revisions.length-1-index %>"><%= raw RDiscount.new(revision.body).to_html %></div>
+        </td>
+        <td><small><a class="btn btn-link" href="/wiki/revert/<%= revision.vid %>" data-confirm="Are you sure?"><i class="fa fa-backward"></i> Revert</a></small></td>
+        <td><small><a class="btn btn-link" href="/wiki/raw/<%= revision.vid %>"><i class="fa fa-code"></i> Raw</a></small></td>
+      </tr>
+    <% end %>
+    </table>
 
-  <hr />
-  
-</div>
-<div class="col-md-7">
-  <div class="btn-group pull-right">
-    <a class="btn btn-link" href="javascript:void(0)" onClick="prev()">&laquo;</a>
-    <a class="btn btn-link" href="javascript:void(0)" onClick="next()">&raquo;</a>
+    <hr />
+    
   </div>
-  <h3 style="margin-top:0;">Comparing revision <span id="older"><%= @node.revisions.length-1 %></span> with revision <span id="newer"><%= @node.revisions.length-2 %></span></h3>
+  <div class="col-md-7">
+    <div class="btn-group pull-right">
+      <a class="btn btn-link" href="javascript:void(0)" onClick="prev()">&laquo;</a>
+      <a class="btn btn-link" href="javascript:void(0)" onClick="next()">&raquo;</a>
+    </div>
+    <h3 style="margin-top:0;">Comparing revision <span id="older"><%= @node.revisions.length-1 %></span> with revision <span id="newer"><%= @node.revisions.length-2 %></span></h3>
 
-  <div class="well" id="diff"></div>
+    <div class="well" id="diff"></div>
 
-  <style>
-    del { background:#faa; }
-    ins { background:#afa; }
-  </style>
+    <style>
+      del { background:#faa; }
+      ins { background:#afa; }
+    </style>
 
-  <script>
+    <script>
 
-  jQuery(document).ready(function() {
-    var newer = <%= @node.revisions.length-1 %>; older = <%= @node.revisions.length-2 %>;
-    $('#diff').html(diffString($('#body_'+older).html(),$('#body_'+newer).html()))
-    $(document).keydown(function(e){ if (e.keyCode == 39) { next() }})
-    $(document).keydown(function(e){ if (e.keyCode == 37) { prev() }})
-    function prev() {
-      if (older > 0) {
-        goto(newer-1,older-1)
-    }}
-    function next() {
-      if (newer < <%= @node.revisions.length-1 %>) {
-        goto(newer+1,older+1)
-    }}
-    function goto(newr,old) {
-      newer = newr
-      older = old
-      $('#newer').html(newer)
-      $('#older').html(older)
+    jQuery(document).ready(function() {
+      var newer = <%= @node.revisions.length-1 %>; older = <%= @node.revisions.length-2 %>;
       $('#diff').html(diffString($('#body_'+older).html(),$('#body_'+newer).html()))
-      $('tr').removeClass('warning')
-      $('#row'+older).addClass('warning')
-      $('#row'+newer).addClass('warning')
-    }
-  });
-
-  </script>
+      $(document).keydown(function(e){ if (e.keyCode == 39) { next() }})
```

Comment:
All this should eventually be moved to a revisions.js file, not inline. But no need in this commit. Thanks!